src/main/java/com/microsoft/sqlserver/jdbc/Column.java

```diff
@@ -189,6 +191,12 @@ Object getValue(JDBCType jdbcType, InputStreamGetterArgs getterArgs, Calendar ca
             SQLServerStatement statement) throws SQLServerException {
         Object value = getterDTV.getValue(jdbcType, typeInfo.getScale(), getterArgs, cal, typeInfo, cryptoMetadata,
                 tdsReader, statement);
+        if (jdbcType == JDBCType.VECTOR && value == null) {
+            int precision = typeInfo.getPrecision(); 
+            VectorDimensionType scale = typeInfo.getScale() == 4 ? VectorDimensionType.F32 : VectorDimensionType.F16;

```

Comment:
Could we move all hard-coded values to constant variables with intuitive and descriptive names?